# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/DocumentChannel.cpp
# Commit: 018034204705
# Full Hash: 018034204705f9e8f95d5a7afbdbc36248fd4a40
# Author: Dan Glastonbury <dan.glastonbury@gmail.com>
# Date: 2020-07-29 09:49:32
# Description:
#   Bug 1654764: Reduce DocumentChannel::GetContentType() crash to error. r=mattwoodrow
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D85100
# ==============================================================================

diff -r 0067bb1fb8e4 -r 018034204705 netwerk/ipc/DocumentChannel.cpp
--- a/netwerk/ipc/DocumentChannel.cpp	Wed Jul 29 02:33:54 2020 +0000
+++ b/netwerk/ipc/DocumentChannel.cpp	Wed Jul 29 03:13:31 2020 +0000
@@ -353,7 +353,7 @@
 }
 
 NS_IMETHODIMP DocumentChannel::GetContentType(nsACString& aContentType) {
-  MOZ_CRASH("If we get here, something is broken");
+  NS_ERROR("If we get here, something is broken");
   return NS_ERROR_NOT_IMPLEMENTED;
 }
 
